src/utils.js

```diff
@@ -10,30 +10,53 @@ export function getNetworkName(chainId) {
 
 export function rpcResult(response) {
   // Some providers don’t wrap the response
-  if (response && response.jsonrpc) {
+  if (response && 'jsonrpc' in response) {
     if (response.error) {
       throw new Error(response.error)
     }
-    return response.result
+    return response.result || ''
   }
-  return response
+  return response || ''
```

Comment:
Would it be better to explicitly return `null` in this case?

Sometimes the ETH JSONRPC returns `null`, e.g. when it can't find a block.

Reply:
Cool let’s do this then! :+1:

Reply:
Done in https://github.com/aragon/use-wallet/pull/21/commits/564a9a90e435fe2ebcd87c927fd1c2e277113573